ethcore/src/service.rs

```diff
@@ -146,6 +148,32 @@ impl ClientService {
 	pub fn add_notify(&self, notify: Arc<ChainNotify>) {
 		self.client.add_notify(notify);
 	}
+
+	#[cfg(feature="stratum")]
+	/// Runs stratum if feature is on
+	pub fn stratum_notifier(cfg: &StratumOptions, miner: Weak<Miner>, client: Weak<Client>) -> Result<Box<miner::NotifyWork>, ()> {
+		use miner::Stratum;
+		use miner::NotifyWork;
```

Comment:
it is there because of the conditional compilation, to avoid warnings when the feature is off